intf/cnsr_intf.lua

```diff
@@ -131,6 +131,9 @@ actions.hide.update= function() check_deactivate(actions.hide) end
 function actions.hide.activate()
 	actions.hide.hide_filter = vlc.object.vout()
 	vlc.var.create(actions.hide.hide_filter, "contrast", 0)
+	vlc.var.create(actions.hide.hide_filter, "brightness", 0)
+	vlc.var.create(actions.hide.hide_filter, "saturation", 0)
+	vlc.var.create(actions.hide.hide_filter, "gamma", 0)
```

Comment:
Did you really have to change **all** of these values?

Reply:
the gamma doesn't have to be changed, the others needs to be changed(if one of them is on you might see something).
my thought with gamma was that it's better safe than sorry, so just in case it will change, we should cover this.